build.rs

```diff
@@ -0,0 +1,53 @@
+fn main() {
+    manage_docs();
+}
+
+#[cfg(any(feature = "embed-lgpl-docs", feature = "purge-lgpl-docs"))]
+fn manage_docs() {
+    extern crate lgpl_docs;
+    const PATH: &str = "src";
+    const IGNORES: &[&str] = &[
+        "lib.rs",
+        "prelude.rs",
+        "signal.rs",
+        "boxed.rs",
+        "byte_array.rs",
+        "bytes.rs",
+        "char.rs",
+        "clone.rs",
+        "enums.rs",
+        "error.rs",
+        "gobject/mod.rs",
+        "log.rs",
+        "main_context.rs",
+        "main_context_channel.rs",
+        "main_context_futures.rs",
+        "object.rs",
+        "send_unique.rs",
+        "shared.rs",
+        "source.rs",
+        "source_futures.rs",
+        "string.rs",
+        "subclass/boxed.rs",
+        "subclass/interface.rs",
+        "subclass/mod.rs",
+        "subclass/object.rs",
+        "subclass/simple.rs",
+        "subclass/types.rs",
+        "translate.rs",
+        "types.rs",
+        "utils.rs",
+        "value.rs",
+        "variant.rs",
+        "variant_dict.rs",
+        "variant_type.rs",
+        "wrapper.rs",
+    ];
+    lgpl_docs::purge(PATH, IGNORES);
+    if cfg!(feature = "embed-lgpl-docs") {
+        lgpl_docs::embed(lgpl_docs::Library::Glib, PATH, IGNORES);
```

Comment:
There's also `src/auto/gobject` from `Gir_GObject.toml` btw, we have two libraries in one here

Reply:
Is it safe to call embed twice I wonder?

Reply:
For different libraries and paths, probably? One would be `src/auto` and the other `src/auto/gobject`

Reply:
As far as I can tell it's all relative to `src/` but maybe @GuillaumeGomez can help here?

Reply:
@GuillaumeGomez ?

Reply:
You gave it `src`, so it shouldn't run on `build.rs`...

Reply:
It's not `build.rs` we're interested in, it's the ability to deal with the fact we have *two* Gir sets here (gobject and glib) to merge docs from.

Reply:
Oh. I'm not sure `rustdoc-stripper` allows to give multiple doc files at once...

Reply:
But can it be called twice with different docs? It would have to handle in this case code to be in `src/*.rs` and `src/auto/gobject/*.rs` instead of `src/*.rs` and `src/auto/*.rs`.

Reply:
It goes through throuh the folder and sub-folders and reads all the `.rs` files. I could add an option to pass files instead of a folder I guess. Would it work out for you?

Reply:
Honestly I cargo-culted this into glib, so you're best placed to work out the best approach.  I'm happy to drop this PR, or to let you take it over too.